Makefile

```diff
@@ -400,7 +403,7 @@ run-client-fullmap: ## Runs the webapp with source code mapping (slower; better
 
 	cd $(BUILD_WEBAPP_DIR) && $(MAKE) run-fullmap
 
-run: run-server run-client ## Runs the server and webapp.
+run: check-prereqs run-server run-client ## Checks prerequisites, Runs the server and webapp.
```

Comment:
I don't think it's really necessary to add "Checks prerequisites" to the documentation. That's something that can happen transparently.

Reply:
I've removed that